pkgs/top-level/all-packages.nix

```diff
@@ -24560,6 +24560,8 @@ with pkgs;
     inherit (darwin.apple_sdk.frameworks) PCSC;
   };
 
+  openpgl = callPackage ../development/libraries/openpgl { };
```

Comment:
Can you use the new by-name structure https://github.com/NixOS/nixpkgs/tree/master/pkgs/by-name#name-based-package-directories ?

Reply:
The point of the by-name structure is, that you don't need this line anymore. but that does not play well with the other suggestion about the tbb override

Reply:
Need to fix CI checks now.